cmd/cli/metrics_enable.go

```diff
@@ -61,6 +61,10 @@ func newMetricsEnable(out io.Writer) *cobra.Command {
 }
 
 func (cmd *metricsEnableCmd) run() error {
+	if len(cmd.namespaces) == 0 {
```

Comment:
cobra allows you to mark flags are required. It seems like that is the case here.  We could use `rootCmd.MarkFlagRequired("namespace")` in the above `newMetricsEnable`.  This would make it more consistent with other error message I think.

https://cobra.dev/#required-flags

Reply:
+1

Reply:
Thanks, @jsturtevant for the comment I have made the required changes 👍🏽